.env.example

```diff
@@ -18,3 +18,7 @@ ALGOLIA_API_KEY=
 # ALGOLIA_INDEX=
 
 # API_BASE_URL_OVERRIDE=
+
+# Uploadcare variables
+NEXT_PUBLIC_UPLOADCARE_PUBLIC_KEY=97775dfb0ac5a6446bce
+NEXT_PUBLIC_UPLOADCARE_CUSTOM_CDN_DOMAIN=cdn.uc.assets.prezly.com
```

Comment:
Hey @camilb, I've added these two env variables but I believe we also need to add them to the `build-and-push.yml` file, right? So I assume they have to be defined as secrets first?